File: source/SEP3 - tier 1 (Rider)/Model/Rent.cs
using System;

namespace WebApplication7.Model
{
    public class Rent
    {
        public int GameId { get; set; }
        public String User { get; set; }
        public long TimeFrom
        {
            get => (RentFrom.Ticks - new DateTime(1970,1, 1).Ticks) / 10000;
            set => RentFrom = new DateTime(value * 10000 + new DateTime(1970, 1, 1).Ticks);
        }
        public long TimeUntil
        {
            get => (RentUntil.Ticks - new DateTime(1970,1, 1).Ticks) / 10000;
            set => RentUntil = new DateTime(value * 10000 + new DateTime(1970, 1, 1).Ticks);
        }
        public DateTime RentFrom { get; set; }
        public DateTime RentUntil { get; set; }

        public override string ToString()
        {
            return "GameId: " + GameId + "\nUser: " + User + "\nRentFrom: " +  RentFrom + "\nRentUntil: " + RentUntil;
        }
    }
}

File: source/SEP3 - tier 3 (IntelliJ)/src/main/java/model/Rent.java
package model;

/**this class serves as a Data Object for Rent*/
public class Rent
{

    /**id of the game which this renting is associated with*/
    public int GameId;
    /**name of the user which this renting is associated with*/
    public String User;
    /**datetime when the game is rented from*/
    public long TimeFrom;
    /**datetime when the game is rented until*/
    public long TimeUntil;

    /**initialises all instance variables*/
    public Rent(int gameId, String user, long timeFrom, long timeUntil)
    {
        GameId = gameId;
        User = user;
        TimeFrom = timeFrom;
        TimeUntil = timeUntil;
    }

    public String toString()
    {
        return GameId + " " + User + " " + TimeFrom + " " + TimeUntil;
    }
}
